### PR TITLE
chore(routes): Implement python routes for new stories & feedback pages

### PR DESCRIPTION
**Pull request by ryan953**

### CODE REVIEW

**Review by JoshFerge — APPROVED** [review]

**Comment by evanpurkhiser** [comment]
I think we can get away without this, we can just build these in vercel.

File: src/sentry/web/urls.py [hunk 1]
```diff
@@ -419,6 +419,12 @@
         react_page_view,
         name="sentry-admin-overview",
     ),
+    # Story book
+    re_path(
```
Comment (evanpurkhiser) [inline src/sentry/web/urls.py:423]: Let's remove these. We can build this in Vercel only.

**Review by evanpurkhiser — CHANGES_REQUESTED** [review]
Let's remove the storybook change

**Comment by ryan953** [comment]
> Let's remove the storybook change

Ok let's put that up quickly. Because right now I'm blocked from copy+pasting links to people. 

So I'd like to be unblocked, even for a day, and then we can remove this code afterwards

**Review by evanpurkhiser — APPROVED** [review]

**Comment by evanpurkhiser** [comment]
sgtm

**Pull request merged**